src/extension.ts

```diff
@@ -164,6 +164,18 @@ export function activate(context: vscode.ExtensionContext) {
             return configDiagnosticProvider.updateDiagnostics(document, diagCollection);
         });
 
+    initCommandAsync(context, outputChannel,
+        "azure-iot-edge.showGallery",
+        async (): Promise<void> => {
+          return edgeManager.loadWebView(outputChannel);
+        });
+
+    initCommandAsync(context, outputChannel,
+      "azure-iot-edge.initializeSample",
```

Comment:
is this command registered in package.json?

Reply:
not needed and removed